### ISSUE TITLE
gqrx shouldn't exit with SIGABRT on user errors

### ISSUE THREAD

**Issue by l29ah** [csete/gqrx#675]
Say,
```
[blah]
Using device #0 Generic RTL2832U OEM
usb_open error -3
Please fix the device permissions, e.g. by installing the udev rules file rtl-sdr.rules
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open rtlsdr device.
zsh: abort (core dumped)  gqrx
```
shouldn't happen as it is not a programming error, yet it produces an useless core dump.

**Comment by csete** [comment]
This is a weakness of the architecture, but please do send a pull request if you think it can be fixed in a clean way.

**Comment by argilo** [comment]
This issue can be closed, since it was addressed in 34333e78b2e565745b6d62bf0efcc57cc3b1988f.

**Comment by l29ah** [comment]
Thanks.
